tests/unit_tests/test_load_forward_model.py

```diff
@@ -110,6 +111,41 @@ def test_load_inconsistent_time_map_summary(caplog):
     assert loaded == 1
 
 
+@pytest.mark.usefixtures("copy_snake_oil_case_storage")
+def test_datetime_2500():
+    """
+    Test that we are able to work with dates past year 2263 in summary files.
+
+    Ref: https://github.com/pydata/xarray/issues/7493
+
+    """
+    cwd = os.getcwd()
+
+    # Get rid of GEN_DATA as we are only interested in SUMMARY
+    with fileinput.input("snake_oil.ert", inplace=True) as fin:
+        for line in fin:
+            if line.startswith("GEN_DATA"):
+                continue
+            print(line, end="")
+
+    facade = LibresFacade.from_config_file("snake_oil.ert")
```

Comment:
I guess we should avoid LibresFasade here in light of removing it in https://github.com/equinor/ert/issues/6687

Reply:
We still need it for `facade.load_from_forward_model` below

Reply:
might be a stupid question, but why? I just tried and the test is passing without this line as well

Reply:
I guess you want to check that it can be loaded without errors, right?

Reply:
Your observations are correct. I have added another assert to check that we actually load whats written to the runpath